Detectors/AOD/include/AODProducerWorkflow/AODProducerWorkflowSpec.h

```diff
@@ -351,6 +355,48 @@ class AODProducerWorkflowDPL : public Task
 
   // helper for trd pattern
   uint8_t getTRDPattern(const o2::trd::TrackTRD& track);
+
+  o2::emcal::EventHandler<o2::emcal::Cell>* mEventHandler = nullptr; ///< Pointer to the event builder for emcal cells
```

Comment:
could you call this `mCaloEventHandler` or similar? Just `mEventHandler` looks a bit too generic.

Reply:
renamed to mCaloEventHandler